app/models/c100_application.rb

```diff
@@ -63,4 +63,8 @@ def mark_as_completed!
       CompletedApplicationsAudit.log!(self)
     end
   end
+
+  def online_submission?
```

Comment:
You are overriding an already perfectly fine method in the `ApplicationInquiryMethods`.

Reply:
addressed